assets/css/order-styles.css

```diff
@@ -68,12 +68,46 @@
 }
 
 .dashicons-edit,
-.dashicons-trash {
+.dashicons-trash,
```

Comment:
These should all be more specific to the WCPDF data panel

Reply:
Solved with: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/commit/d66f3cd72ed257f979294e997ecddeaa5b581da8